api/integrations/github/views.py

```diff
@@ -86,9 +120,13 @@ def perform_create(self, serializer):
         serializer.save(github_configuration_id=github_configuration_id)
 
     def get_queryset(self):
-        return GithubRepository.objects.filter(
-            github_configuration=self.kwargs["github_pk"]
-        )
+        github_pk = self.kwargs.get("github_pk")
+        if github_pk is not None:
```

Comment:
nit: This method will cause an internal server error if `github_pk` is `None`. This renders the elaborate validation below incomplete.

Reply:
Done